cmd/server.go

```diff
@@ -57,11 +58,12 @@ var serverCmd = &cobra.Command{
 		// in the future, we may support other device manager types
 		var mgr driver.DeviceManager
 		maxSizes := common.MaxSizes{
-			MaxLogSize:      maxLogSize,
-			MaxInfoSize:     maxInfoSize,
-			MaxMetricSize:   maxMetricSize,
-			MaxRequestsSize: maxRequestsSize,
-			MaxAppLogsSize:  maxAppLogsSize,
+			MaxLogSize:         maxLogSize,
+			MaxInfoSize:        maxInfoSize,
+			MaxMetricSize:      maxMetricSize,
+			MaxRequestsSize:    maxRequestsSize,
+			MaxFlowMessageSize: maxFlowRecordSize,
```

Comment:
Why do we call the var `maxFlowRecordSize` but the property `MaxSizes.MaxFlowMessageSize`? Is there a distinction between "message" size and "record" size? If there is, we should explain in a comment what the two are; if not, let's unify the naming, otherwise we confuse people.